src/CfgEditor/CfgEditorPanel.ts

```diff
@@ -111,7 +111,80 @@ export class CfgEditorPanel implements vscode.CustomTextEditorProvider {
     // Receive message from the webview.
     webviewPanel.webview.onDidReceiveMessage(e => {
       switch (e.type) {
-        // TODO Add more webview modification handlers
+        case 'setParam':
+          if (this._oneConfig[e.section] === undefined) {
+            this._oneConfig[e.section] = {};
+          }
+          if (this._oneConfig[e.section][e.param] === undefined) {
+            this._oneConfig[e.section][e.param] = '';
+          }
+          this._oneConfig[e.section][e.param] = e.value;
+          break;
+        case 'setSection':
+          this._oneConfig[e.section] = ini.parse(e.param);
+          break;
+        case 'updateDocument':
+          let isSame = true;
+          const iniDocument = ini.parse(document.getText());
```

Comment:
Hmm... below code can be extracted to a function. 

(Optional) However, any better way? like using `diff`?

Reply:
For it, of course, rules like whether using `whitespace` or not should be aligned.

Reply:
I got why my idea cannot. the sections and its keys can have own orders. /cc @llFreetimell